lib/nmatrix/math.rb

```diff
@@ -434,7 +490,7 @@ def complex_conjugate(new_stype = self.stype)
   # == Options
   # 
   # * +:for_sample_data+ - Default true. If set to false will consider the denominator for
-  #   population data (i.e. N-1).
+  #   population data (i.e. N, as opposed to N-1 for sample data).
```

Comment:
Don't worry about this for the current PR, but for the future, please don't include changes (even to docs or whitespace) unrelated to the topic of the PR; it makes it harder to review the diff.